bin/supervisor/src/flags/supervisor.rs

```diff
@@ -67,6 +78,64 @@ impl SupervisorArgs {
         })?;
         Ok(dependency_set)
     }
+
+    /// initialise and return the rollup config set.
+    pub async fn init_rollup_config_set(&self) -> Result<RollupConfigSet> {
+        let pattern = self.rollup_config_paths.to_string_lossy();
+        if pattern.is_empty() {
+            return Err(anyhow::anyhow!("rollup_config_paths pattern is empty"));
+        }
+
+        let l1_url = self
+            .l1_rpc
+            .parse()
+            .with_context(|| format!("Failed to parse L1 RPC URL '{}'", &self.l1_rpc))?;
+        let provider = RootProvider::<Ethereum>::new_http(l1_url);
+
+        let mut rollup_config_set = RollupConfigSet::default();
+        for entry in glob(&pattern)? {
+            let path = entry?;
+            let mut file = File::open(&path).await.with_context(|| {
+                format!("Failed to open rollup config file '{}'", path.display())
+            })?;
+            let mut contents = String::new();
+            file.read_to_string(&mut contents).await.with_context(|| {
+                format!("Failed to read rollup config file '{}'", path.display())
+            })?;
+
+            let rollup_config: kona_genesis::RollupConfig = serde_json::from_str(&contents)
```

Comment:
no need for this long path here

Reply:
Fixed this in the [source PR](https://github.com/op-rs/kona/pull/1953)